<launch>
    <group ns="move_base_simple">
     <rosparam file="$(find chaotic_CPP)/config/ArnoldLogistic.yaml" />  
    </group>

    <rosparam file="$(find chaotic_CPP)/config/mapzoner.yaml" />

    <node pkg="chaotic_CPP" type="ArnoldLogistic_coverage.py" name="controlled_chaotic_trajectory_planner"    ns="move_base_simple" output="screen"/>

   <node pkg="chaotic_CPP" type="map_zoner.py" name="mapzoner" output="screen"/>
    
</launch> 
